example/src/test/java/com/alipay/sofa/rpc/http2/Http2WithSSLClientMain.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.sofa.rpc.http2;
+
+import com.alipay.sofa.rpc.config.ApplicationConfig;
+import com.alipay.sofa.rpc.config.ConsumerConfig;
+import com.alipay.sofa.rpc.context.RpcRuntimeContext;
+import com.alipay.sofa.rpc.log.Logger;
+import com.alipay.sofa.rpc.log.LoggerFactory;
+import com.alipay.sofa.rpc.protobuf.EchoRequest;
+import com.alipay.sofa.rpc.protobuf.EchoResponse;
+import com.alipay.sofa.rpc.protobuf.Group;
+import com.alipay.sofa.rpc.protobuf.ProtoService;
+
+/**
+ *
+ * @author <a href="mailto:466178395@qq.com">LiHao</a>
+ */
+public class Http2WithSSLClientMain {
+    /**
+     * slf4j Logger for this class
+     */
+    private final static Logger LOGGER = LoggerFactory.getLogger(Http2WithSSLClientMain.class);
+
+    public static void main(String[] args) {
+        ApplicationConfig application = new ApplicationConfig().setAppName("test-client");
```

Comment:
You put public&private key to  `/example/src/test/resources/privatekey.key` and `/example/src/test/resources/selfSigned.crt`, and can set property here, like 
```java
        System.setProperty("ssl", "true");
        String codebase = ReflectUtils.getCodeBase(Http2WithSSLClientMain.class);
        System.setProperty("certificate_path", codebase + "/selfSigned.crt");
```

Reply:
i hava modified.